src/io/flutter/vmService/VMServiceManager.java

```diff
@@ -644,7 +644,9 @@ public void received(JsonObject object) {
   }
 
   private CompletableFuture<String> getFlutterViewId() {
-    return getFlutterViewsList().thenApplyAsync((JsonElement element) -> {
+    return getFlutterViewsList().exceptionally(exception -> {
+      throw new RuntimeException(exception.getMessage());
```

Comment:
Just confirming, when we complete exceptionally here (and below on the `ret.completeExceptionally()` line) - is that exception displayed to users? Or do we fall back on assuming 60hz?

(I'd want to make sure that these issues are not surfaced to the users as exceptions)

Reply:
The exception is not displayed to users - we fall back to 60 and log what happened as a warning.